src/main/java/seedu/address/storage/AddressBookStorage.java

```diff
@@ -19,16 +19,16 @@ public interface AddressBookStorage {
 
     /**
      * Returns AddressBook data as a {@link ReadOnlyAddressBook}.
-     *   Returns {@code Optional.empty()} if storage file is not found.
+     * Returns {@code Optional.empty()} if storage file is not found.
+     *
      * @throws DataLoadingException if the data in storage is not in the expected format.
```

Comment:
Doesn't this need updating as well?

Reply:
I missed out this when doing the rebasing to shift the documentation change to the 2nd commit.